app/views/teaching_events/index.html.erb

```diff
@@ -6,26 +6,13 @@
       Find out more about getting into teaching at a free event where you can get all your questions answered.
```

Comment:
@philspray This section feels a bit wordy, particularly on mobile, given there are only two event types now?

Should we shorten the first sentence / get rid of it / reword the other sections - or is it OK as is?

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/c3da57c6-2cf0-400f-bfd9-5435736f5c44)

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/89dee5b6-38d2-4da2-98c0-690a0b846f9b)

Paging @EllieNodder for her view as she was around when we first added this intro section/accordion!